lib/Gateway.js

```diff
@@ -666,7 +666,8 @@ function getEntityName (node, valueId, cfg, entityTemplate) {
     label = valueId.label
   }
   return entityTemplate
-    .replace(/%n/g, node.name)
+    .replace(/%ann/g, nodeName)
```

Comment:
I prefer the old way, this introduce just confusiuon about the difference between the auto generated and the default node name, it's clear to us but not to the end user

Reply:
this is removed